virttest/utils_net.py

```diff
@@ -3092,13 +3092,17 @@ def get_linux_ipaddr(session, nic):
     """
     cmd = "ifconfig %s || ip address show %s" % (nic, nic)
     out = session.cmd_output(cmd)
+    inet_regex = "inet (addr:)?(\d+.\d+.\d+.\d+)"
     try:
-        inet = re.findall("inet (\S+)", out, re.M)[0]
-    except IndexError:
+        inet = re.search(inet_regex, out, re.M).groups()[1]
+    except Exception:
         inet = None
-    inet6 = re.findall("inet6 (\S+)", out, re.M)
-    if not inet6:
-        inet6 = [None]
+
+    inet6_regex = "inet6 (addr:)?(\S+)"
```

Comment:
Should add a space behind "addr:". Below is an example of ip info on rhel6 host.
switch    Link encap:Ethernet  HWaddr 00:22:19:30:0D:20  
          inet addr:10.xx.xx.xx  Bcast:10.xx.xx.xx  Mask:255.255.252.0
          inet6 addr: xxxx::xxx:xxxx:xxxx:xxx/64 Scope:Link